desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -0,0 +1,137 @@
+---
+description: settings management for desktop
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure Settings Management
+--- 
+
+>**Note**
+>
+>Settings Management is available to Docker Business customers only. 
+
+This page contains information for admins on how to configure Settings Management to specify and lock configuration parameters to create a standardized Docker Desktop environment across the organization.
+
+Settings Management is designed specifically for organizations who don’t give developers root access to their machines.
+
+### Prerequisites
+
+- [Download and install Docker Desktop 4.13.0 or later](../../release-notes.md).
+- As an admin, you need to [configure a registry.json to enforce sign-in](../../../docker-hub/configure-sign-in.md). This is because this feature requires a Docker Business subscription and therefore your Docker Desktop users must authenticate to your organization for this configuration to take effect.
+
+### Step one: Create the `admin-settings.json` file and save it in the correct location
+
+You can either use the `--admin-settings` installer flag on [macOS](../../install/mac-install.md#install-from-the-command-line) or [Windows](../../install/windows-install.md#install-from-the-command-line) to automatically create the `admin-settings.json` and save it in the correct location, or set it up manually. 
+
+To set it up manually:
+1. Create a new, empty JSON file and name it `admin-settings`.
+2. Save the `admin-settings.json` file on your developers' machines in the following locations:
+
+    - Mac: `/Library/Application\ Support/com.docker.docker/admin-settings.json`
+    - Windows: `C:\ProgramData\DockerDesktop\admin-settings.json`
+    - Linux: `/usr/share/docker-desktop/admin-settings.json`
+
+    By placing this file in the above protected directories, end users are unable to modify it.
+
+    >**Note**
+    >
+    > It is assumed that you have the ability to push the `admin-settings.json` settings file to the locations specified above through a device management software such as [Jamf](https://www.jamf.com/lp/en-gb/apple-mobile-device-management-mdm-jamf-shared/?attr=google_ads-brand-search-shared&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhEXjayUAi8FHHv1JJitFPb47C_q_RCySTmF86twF1qJc_6GST-YDmhoCuJsQAvD_BwE).
```

Comment:
we should mention that you can use command line flag also here now, no?

Reply:
I added it in the lines above. second paragraph under 'How do I set up and enforce Settings Management'